### PR TITLE
Optimization checklist as an HTML table

### PR DESCRIPTION
**Pull request by stoyan**
* kept the image as an option to download
* reused styles from the history table
* used as opportunity to show off some Blade templating

There's definitely room for improvement both in terms of UI (e.g. old pixel-y icons) and optimization advice, but it's a start a move away from image generation

### CODE REVIEW

**Comment by stoyan** [comment]
Before: https://www.webpagetest.org/result/221007_AiDc8P_FYA/1/performance_optimization/
After:

https://user-images.githubusercontent.com/51308/196517332-be544c3c-bfc9-4d90-a702-aaca39b0430b.mov

File: www/resources/views/snippets/optimizationchecklist.blade.php [hunk 1]
```diff
@@ -1 +1,32 @@
...
+                @else
+                    N/A
+                @endif
+            </th>
+            @endforeach
+        </tr>
+    </thead>
+    <tbody>
+        @foreach ($rows as $row)
+        <tr class="table-cell-{{ $row['color'] ?? ($loop->even ? 'even' : 'odd')}}">
```
Comment (jefflembeck) [inline www/resources/views/snippets/optimizationchecklist.blade.php:20]: Where does `$loop` come from?

**Comment by stoyan** [comment]
https://laravel.com/docs/9.x/blade#the-loop-variable
Blade magic

File: www/resources/views/snippets/optimizationchecklist.blade.php (hunk 1 above)
Comment (stoyan) [reply www/resources/views/snippets/optimizationchecklist.blade.php:20]: `$loop` is some Blade magic that helps working with loops
https://laravel.com/docs/9.x/blade#the-loop-variable

**Comment by tkadlec** [comment]
Yay for HTML! And templating!

**Pull request merged**